tests/writer_util.py

```diff
@@ -159,6 +159,53 @@ def _convert_clob_pairs(clob_pairs):
         (_DT(2016, 1, 1, 12, 34, 12, tzinfo=OffsetTZInfo()), b'2016-01-01T12:34:12.000000Z'),
         (_DT(2016, 1, 1, 12, 34, 12, tzinfo=OffsetTZInfo(timedelta(hours=-7))),
             b'2016-01-01T12:34:12.000000-07:00'),
+        (timestamp(year=1, month=1, day=1, precision=TimestampPrecision.DAY), b'0001-01-01T'),
```

Comment:
These were copied from `SIMPLE_SCALARS_MAP_BINARY` below and given expected text values. There's no reason to have this coverage for binary but not text.